blog/a-very-useful-data-structure-linked-lists.md

```diff
@@ -0,0 +1,97 @@
+---
+slug: a-very-useful-data-structure-linked-lists
+title: A Very Useful Data Structure - Linked Lists
+authors: [Aryan-Jain]
```

Comment:
write your `Aryan-Jain` name and basic info in the `blog/authors.yml` file like https://github.com/ajay-dhangar/algo/blob/7814b98e8e0993e3b52f573f23810bb19afd453c/blog/authors.yml#L32